src/redux/sliceResources/resources.ts

```diff
@@ -0,0 +1,72 @@
+import { createUrlPath } from '~/utils/helper-functions'
+import { URLs } from '~/constants/request'
+import { appApi } from '~/redux/apiSlice'
+
+import { Attachment, ItemsWithCount, Lesson, LessonData } from '~/types'
+
+export interface LessonsQueryArgs {
+  limit: number
+  skip: number
+  sort: object
+  title: string
+  categories: string[]
+}
+type DeleteLessonArgs = string
+
+interface EditLessonArgs {
+  id?: string
+  title: string
+  description: string
+  content: string
+  attachments: Attachment[]
+  category: string | null
+}
+
+export const apiWithTag = appApi.enhanceEndpoints({
+  addTagTypes: ['Lessons']
+})
+
+export const resourcesApi = apiWithTag.injectEndpoints({
+  endpoints: (builder) => ({
+    getLessons: builder.query<ItemsWithCount<Lesson>, LessonsQueryArgs>({
+      query: (params) => {
+        return {
```

Comment:
return it immediately:

query: (params) => ({ url: createUrlPath(URLs.resources.lessons.get),
          params
        })

Reply:
fixed